sdk/translation/azure-ai-translation-document/src/test/java/com/azure/ai/translation/document/DocumentTranslationClientTestBase.java

```diff
@@ -107,16 +105,12 @@ String getStorageName() {
             : Configuration.getGlobalConfiguration().get("DOCUMENT_TRANSLATION_STORAGE_NAME");
     }
 
-    private String getConnectionString() {
-        return interceptorManager.isPlaybackMode()
-            ? "DefaultEndpointsProtocol=https;AccountName=dummyAccount;AccountKey=xyzDummy;EndpointSuffix=core.windows.net"
-            : Configuration.getGlobalConfiguration().get("DOCUMENT_TRANSLATION_CONNECTION_STRING");
-    }
-
     BlobContainerClient getBlobContainerClient(String containerName) {
+        String endpoint = String.format("https://%s.blob.core.windows.net", getStorageName());
         BlobContainerClientBuilder blobContainerClientBuilder = new BlobContainerClientBuilder()
+            .endpoint(endpoint)
             .containerName(containerName)
-            .connectionString(getConnectionString());
+            .credential(new DefaultAzureCredentialBuilder().build());
```

Comment:
Replace this with below logic: 

```java
.credential(getIdentityTestCredential());
```
Logic for above method.
```java
    public TokenCredential getIdentityTestCredential(InterceptorManager interceptorManager) {
        if (interceptorManager.isPlaybackMode()) {
            return new MockTokenCredential();
        }

        Configuration config = Configuration.getGlobalConfiguration();

        ChainedTokenCredentialBuilder builder = new ChainedTokenCredentialBuilder()
                .addLast(new EnvironmentCredentialBuilder().build())
                .addLast(new AzureCliCredentialBuilder().build())
                .addLast(new AzureDeveloperCliCredentialBuilder().build());


        String serviceConnectionId = config.get("AZURESUBSCRIPTION_SERVICE_CONNECTION_ID");
        String clientId = config.get("AZURESUBSCRIPTION_CLIENT_ID");
        String tenantId = config.get("AZURESUBSCRIPTION_TENANT_ID");
        String systemAccessToken = config.get("SYSTEM_ACCESSTOKEN");

        if (!CoreUtils.isNullOrEmpty(serviceConnectionId)
                && !CoreUtils.isNullOrEmpty(clientId)
                && !CoreUtils.isNullOrEmpty(tenantId)
                && !CoreUtils.isNullOrEmpty(systemAccessToken)) {

            AzurePipelinesCredential azurePipelinesCredential = new AzurePipelinesCredentialBuilder()
                    .systemAccessToken(systemAccessToken)
                    .clientId(clientId)
                    .tenantId(tenantId)
                    .serviceConnectionId(serviceConnectionId)
                    .build();

            builder.addLast(trc -> azurePipelinesCredential.getToken(trc).subscribeOn(Schedulers.boundedElastic()));
        }

        builder.addLast(new AzurePowerShellCredentialBuilder().build());


        return builder.build();
    }
```

Reply:
Thank you. Updated